zcash_client_backend/src/data_api.rs

```diff
@@ -541,6 +541,71 @@ pub struct SpendableNotes<NoteRef> {
     orchard: Vec<ReceivedNote<NoteRef, orchard::note::Note>>,
 }
 
+/// A request for transaction data enhancement, spentness check, or discovery
+/// of spends from a given transparent address within a specific block range.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum TransactionDataRequest {
+    /// Information about the chain's view of a transaction is requested.
+    ///
+    /// The caller evaluating this request on behalf of the wallet backend should respond to this
+    /// request by determining the status of the specified transaction with respect to the main
+    /// chain; if using `lightwalletd` for access to chain data, this may be performed interpreting
+    /// the results of the [`GetTransaction`] RPC method. It should then call
```

Comment:
```suggestion
    /// chain; if using `lightwalletd` for access to chain data, this may be achieved by
    /// interpreting the results of the [`GetTransaction`] RPC method. It should then call
```